lib/playground.dart

```diff
@@ -187,7 +191,12 @@ class Playground {
 
     keys.bind('ctrl-s', _handleSave);
     keys.bind('ctrl-enter', _handleRun);
-    keys.bind('f1', _handleHelp);
+    keys.bind('f1', () => _toggleDocTab());
+    document.onKeyUp.listen((e) => _handleHelp());
+    document.onClick.listen((e) => _handleHelp());
```

Comment:
@devoncarew Hm.. I did really like it that the documentation automatically updates also if you scroll through the  completion popup, as in here:
![](http://i.imgur.com/GFrExbX.gif﻿)

What about showing the documentation onClick and onKeyDown if the key is an arrow key ? So that this feature will keep working.